tests/TestCase/Database/Schema/MysqlSchemaDialectTest.php

```diff
@@ -51,268 +51,6 @@ protected function _needsConnection(): void
         $this->skipIf(!str_contains($config['driver'], 'Mysql'), 'Not using Mysql for test config');
     }
 
-    /**
-     * Data provider for convert column testing
-     *
-     * @return array
-     */
-    public static function convertColumnProvider(): array
-    {
```

Comment:
I don't think we have this kind of coverage on column type conversion elsewhere in our tests. It would be ideal to not lose coverage of this behaviour entirely. I can open a pull request later this week to address the coverage gap and use these providers in new tests.

Reply:
> I don't think we have this kind of coverage on column type conversion elsewhere in our tests. It would be ideal to not lose coverage of this behaviour entirely.

I was concerned about the same as I noticed that the removed data providers were only used by the deprecated methods and not the new ones.

> I can open a pull request later this week to address the coverage gap and use these providers in new tests.

Okay, in that case I'll merge this.